packages/react-native-codegen/scripts/oss/build.sh

```diff
@@ -48,4 +60,5 @@ else
 
   mv "$TMP_DIR/lib" "$TMP_DIR/node_modules" "$CODEGEN_DIR"
   rm -rf "$TMP_DIR"
+
```

Comment:
I think this extra newline was left in by accident?

Reply:
Yup